components/SingleClassView/Attendance/StudentAttendanceBox.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext, useState } from "react";
+import styles from "./attendance.module.css";
+import { SingleUserAttendance } from "../../../models";
+import moment from "moment";
+import { CustomLoader } from "../../util/CustomLoader";
+import { AttendanceTypes, CreateAttendance } from "../,,/../../../models";
+import { AttendanceRow } from "./AttendanceRow";
+import { APIContext } from "../../../context/APIContext";
+
+type StudentAttendanceBoxProps = {
+  studentAttendance: SingleUserAttendance[];
+};
+
+const StudentAttendanceBox: React.FC<StudentAttendanceBoxProps> = ({
```

Comment:
commenting on what this student attendance box component is

Reply:
The student attendance box has the render with the date and disabled attendance buttons.